Mathlib/Algebra/DualQuaternion.lean

```diff
@@ -24,7 +24,6 @@ rigid motions in 3D space can be represented by dual quaternions of unit length.
 * <https://en.wikipedia.org/wiki/Dual_quaternion>
 -/
 
-
 variable {R : Type*} [CommRing R]
```

Comment:
This is already a very big PR. I'd recommend against making whitespace changes in files you're not otherwise changing in this PR.

Reply:
Something had changed inbetween ; I'm sorry.